pkgs/data/misc/ddccontrol-db/default.nix

```diff
@@ -1,31 +1,25 @@
 { lib, stdenv
-, autoconf
-, automake
-, libtool
+, autoreconfHook
 , intltool
 , fetchFromGitHub
 }:
 
 stdenv.mkDerivation rec {
   pname = "ddccontrol-db";
-  version = "20220903";
+  version = "20230124";
 
   src = fetchFromGitHub {
     owner = "ddccontrol";
-    repo = "ddccontrol-db";
+    repo = pname;
     rev = version;
-    sha256 = "sha256-OmroJ2AEQdELE2Wuk1qPXzp+/9RUUULrEAdEpmXrJxw=";
+    sha256 = "sha256-coKdlV2DCZ0S3hg9bQEfE9/FaUbFX5w6IMj/kAqERic=";
   };
 
   preConfigure = ''
     ./autogen.sh
   '';
```

Comment:
This may or may not be still required.
Upstream is doing something beyond by knowledge in autogen.sh: https://github.com/ddccontrol/ddccontrol-db/blob/5e5d382ac6fd0a57f4694cf13c7e829e7168d3fa/autogen.sh#L14
Please double check.

Reply:
It seems to work with and without it, seems the bug-fix for sed you mention is from 2017 - so I would go without it we will see on further updates if the file will get relevant?

Reply:
Fine.